public/app/features/alerting/routes.tsx

```diff
@@ -109,6 +111,7 @@ export function getAlertingRoutes(cfg = config): RouteDescriptor[] {
       roles: evaluateAccess([
         AccessControlAction.AlertingNotificationsWrite,
         AccessControlAction.AlertingNotificationsExternalWrite,
+        ...PERMISSIONS_CONTACT_POINTS,
```

Comment:
With this addition, I 'viewer' user can go to this route (tested locally)

Reply:
Yes good catch, thank you! I've split the permissions now and pushed a change